.github/workflows/module-test.yml

```diff
@@ -8,18 +8,26 @@ on:
   schedule:
     - cron:  '0 6 * * *'
 
+env:
+   DRUPAL_TESTING_TEST_CODING_STYLES: false
+   DRUPAL_TESTING_DATABASE_USER: root
+   DRUPAL_TESTING_DATABASE_PASSWORD: root
+   DRUPAL_TESTING_DATABASE_ENGINE: mysql
+
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        DRUPAL_TESTING_DRUPAL_VERSION: ['~8.8.0', '~8.9.0']
+        DRUPAL_TESTING_DRUPAL_VERSION: ['~8.8.0', '~8.9.0', '~9.0.0']
         PHP: ['7.3', '7.4']
         exclude:
           - DRUPAL_TESTING_DRUPAL_VERSION: '~8.8.0'
             PHP: '7.4'
+          - DRUPAL_TESTING_DRUPAL_VERSION: '~9.0.0'
+            PHP: '7.3'
```

Comment:
D9 on 7.3 should work fine, only 7.2 and lower don't but they're not even tested here?